packages/marshal/NEWS.md

```diff
@@ -1,5 +1,10 @@
 User-visible changes in `@endo/marshal`:
 
+# v0.8.1 (2022-12-23)
+
+- Remote objects now reflect methods present on their prototype chain.
+- Serialization errors now serialize.
```

Comment:
Neither of these were changes to the marshal package. But I agree it's good to list them here too.